.github/workflows/maven.yml

```diff
@@ -23,18 +23,20 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest]
-        java: ['11', '17']
 
     steps:
       - uses: actions/checkout@v4
-      - name: Set up JDK ${{ matrix.java }}
+      - name: Set up JDKs
         uses: actions/setup-java@v3
         with:
-          java-version: ${{ matrix.java }}
           distribution: 'temurin'
           cache: 'maven'
-      - name: Build and Test with Java ${{ matrix.java }}
-        run: mvn -B clean verify
+          java-version: |
+              11
+              17
+              21
```

Comment:
This doesn't look like ti's working as expected. I don't see Java 11 or 17 tests being ran.

Reply:
I had given the architecture as `X86` instead of `X64`.  It's not the first time I've made this mistake either. 🥲

Reply:
ha, didn't even catch that part.